perl-lib/OESS/lib/OESS/Workgroup.pm

```diff
@@ -73,6 +73,7 @@ sub to_hash {
     $hash->{name}         = $self->{name};
     $hash->{description}  = $self->{description};
     $hash->{type}         = $self->{type};
+    $hash->{status}       = $self->{status};
```

Comment:
Could you also add status into the `from_hash` function? If it's not passed you can default it to 'active'. This was we can use the results from `OESS::DB::Workgroup::fetch_all` to generate workgroup objects.